templates/layout/sidebar_subscriptions.html.twig

```diff
@@ -1,6 +1,43 @@
+                        {% endif %}
+                        <i class="fa-solid fa-arrow-right"></i>
+                    </a>
+                {% else %}
+                    <a href="#" style="margin-right: .5em;" title="{{ 'subscription_sidebar_move_left'|trans }}" aria-label="{{ 'subscription_sidebar_move_left'|trans }}"
+                        {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SIDEBARS_SAME_SIDE')) is same as constant('App\\Controller\\User\\ThemeSettingsController::TRUE') %}
+                            onclick="fetch('{{ path('theme_settings', {key: constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SIDEBARS_SAME_SIDE'), value: constant('App\\Controller\\User\\ThemeSettingsController::FALSE')}) }}').then(() => document.location.reload())">
+                        {% else %}
+                            onclick="fetch('{{ path('theme_settings', {key: constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SIDEBARS_SAME_SIDE'), value: constant('App\\Controller\\User\\ThemeSettingsController::TRUE')}) }}').then(() => document.location.reload())">
+                        {% endif %}
+                        <i class="fa-solid fa-arrow-left"></i>
+                    </a>
+                {% endif %}
+            {% endif %}
+            {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE')) is not same as 'true' %}
+                <a href="#" title="{{ 'subscription_sidebar_pop_out'|trans }}" aria-label="{{ 'subscription_sidebar_pop_out'|trans }}"
+                   onclick="fetch('{{ path('theme_settings', {key: constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE'), value: constant('App\\Controller\\User\\ThemeSettingsController::TRUE')}) }}').then(() => document.location.reload())">
+                   <i class="fa-solid fa-right-from-bracket"></i>
+                </a>
+            {% else %}
+                <a href="#" title="{{ 'subscription_sidebar_pop_in'|trans }}" aria-label="{{ 'subscription_sidebar_pop_in'|trans }}"
+                   onclick="fetch('{{ path('theme_settings', { key: constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_IN_SEPARATE'), value: constant('App\\Controller\\User\\ThemeSettingsController::FALSE')}) }}').then(() => document.location.reload())">
+                   <i class="fa-solid fa-right-to-bracket"></i>
+                </a>
+            {% endif %}
+        </span>
+    </h3>
+    <div class="subscription-list {{ app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_GENERAL_SHOW_SUBSCRIPTIONS_LARGE')) is same as 'true' ? 'lg' : '' }}">

```

Comment:
I think if you add the class `meta` here, you'll get a lot of the styling to match the other widgets. For example the `.75` opacity and the text color

before:
![image](https://github.com/MbinOrg/mbin/assets/146029455/9932625d-9448-4d40-9adb-fd27370e7f88)

after:
![image](https://github.com/MbinOrg/mbin/assets/146029455/b25740f4-433a-4cb3-a470-c9799de99dfb)